.circleci/config.yml

```diff
@@ -5,48 +5,100 @@ orbs:
   azure-cli: circleci/azure-cli@1.1.0
 
 jobs:
+  build:
+    docker: &python3-7
+      - image: circleci/python:3.7.9
+    steps:
+      - checkout
+      - run:
+          name: install pyodbc deps
+          command: |
+            sudo apt-get update && \
```

Comment:
I think the `&& \` or not needed inside the CI. 

This is a common practice to do with Docker files. They are build up from layers, one for each `RUN` command. You can reduce the number of layers by chaining the bash statements with `&& \`. You can also reduce the size of a layer if you finish with a clean up command. In this case there is not a benefit of doing this.